scripts/omGeoJSON.py

```diff
@@ -42,48 +57,65 @@ def processGeoJSON():
     # Load domain
     print("Loading output file")
     ds = nc.Dataset(domainOutputPath)
-    landmask = ds["LANDMASK"][:]
-    lats = ds["LATD"][:][0]
-    longs = ds["LOND"][:][0]
-    ch4 = ds["OCH4_TOTAL"][:][0][0]
-    maxEmission = np.amax(ch4)
+
+    # There is a better way to do this but this will work for now
+    # Using xarray wasn't straightforward because the layers don't use
+    # the same attribute names, ie mixing x/y and lat/long.
```

Comment:
That needs to be fixed. The naming is incorrect, but the dimensions are the right size which makes it harder to handle than it should

Reply:
Agree, but not within the scope of this change. The code here should continue to work after layer names are changed due to its simplicity, so I'd like to leave it as-is for now.

Reply:
👍